spec/numberformat.html

```diff
@@ -30,9 +30,12 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
```

Comment:
I think we also need to add this change to "Additions and Changes That Introduce Incompatibilities with Prior Editions".